### PR TITLE
Added api-service and skeleton assessments-api-service

### PR DESCRIPTION
**Pull request by jampies**

### CODE REVIEW

File: src/app/app.module.ts.orig [hunk 1]
```diff
@@ -0,0 +1,58 @@
+import { BrowserModule } from '@angular/platform-browser';
```
Comment (Rob0girl) [inline src/app/app.module.ts.orig]: This should be in the git ignore file.

**Review by Rob0girl — CHANGES_REQUESTED** [review]
Move that one file to the git ignore

**Comment by jampies** [comment]
I've added .orig files to gitignore and removed the files. Please review again

**Comment by Rob0girl** [comment]
Looks good

**Pull request merged**